recipes-oim/sota-client/sota-client.bb

```diff
@@ -117,7 +117,11 @@ do_install() {
   install -m 0755 run/sota_sysinfo.sh ${D}${bindir}
 
   install -d ${D}${systemd_unitdir}/system
-  install -c ${S}/run/sota_client.service ${D}${systemd_unitdir}/system
+  if ${@bb.utils.contains('DISTRO_FEATURES', 'sota', 'true', 'false', d)}; then
+    install -c ${S}/run/sota_client_ostree.service ${D}${systemd_unitdir}/system/sota_client.service
+  else
+    install -c ${S}/run/sota_client.service ${D}${systemd_unitdir}/system/sota_client.service
```

Comment:
I'm fine with the changes, but can the fallback case point to `sota_client_yocto.service`? I've removed the `sota_client.service` file in the latest PR since it is no longer obvious where it is used.